hexa/pipelines/management/commands/pipelines_runner.py

```diff
@@ -23,6 +23,30 @@ class PodTerminationReason(Enum):
     DeadlineExceeded = "DeadlineExceeded"
 
 
+def build_run_env(target, env_vars: dict):
```

Comment:
If you call this function in the two cases and you clearly have a if to handle those exact two cases, it's better to leave the code in the corresponding caller function